sip-core/src/test/java/de/ikor/sip/foundation/core/openapi/OpenApiContextPathResolverTest.java

```diff
@@ -0,0 +1,34 @@
+package de.ikor.sip.foundation.core.openapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.PathItem;
+import io.swagger.v3.oas.models.Paths;
+import org.apache.camel.CamelContext;
+import org.junit.jupiter.api.Test;
+
+class OpenApiContextPathResolverTest {
```

Comment:
Could we use SprinBootTest in this case? 
Like start the endpoints and test if they are available under expected path.